doc/source/release-notes/Percona-Server-5.6.26-74.0.rst

```diff
@@ -0,0 +1,72 @@
+ 
+ |TokuDB| Hot Backup has been renamed to |Percona TokuBackup| and it is now `open source <https://github.com/percona/Percona-TokuBackup.git>`_. Source `code <https://github.com/percona/tokudb-backup-plugin>`_ has been integrated into |Percona Server| code as git submodule. 
+
+ *Tokutek Fractal Tree* has been renamed to |Percona FT| and its `source code <https://github.com/percona/PerconaFT.git>`_ has been integrated into |Percona Server| code as git submodule. 
+
+ Google `SNAPPY <http://google.github.io/snappy/>`_ compression/decompression algorithm is now available as :ref:`tokudb_compression` table format.
+
+ |Percona Server| now supports changing the :variable:`server_id` variable :ref:`per session <per_session_server-id>`, by implementing the new :variable:`pseudo_server_id` variable. This feature is also fixing upstream bug :mysqlbug:`35125`.
+
+ |Percona Server| has temporarily disabled savepoints in triggers and stored functions. The reason is that even having fixed bug :bug:`1438990` and bug :bug:`1464468` we have found more cases where savepoints in triggers break binary logging and replication, resulting in server crashes and broken slaves. This feature will be disabled until the above issues are properly resolved.
+
+ ``LOCK BINLOG FOR BACKUP`` can be modified to flush (and redo-log) the current binary log coordinates to |InnoDB|. 
+ 
+ ``innodb_stress`` has been added to the list of default MTR suites. This adds Python and MySQL Python system requirement for default MTR run. The default Python on *CentOS* 5 is too old, python26 and python26-mysqldb have be installed there and made default.
+
+
+Bugs Fixed
+==========
+
+ Querying :table:`GLOBAL_TEMPORARY_TABLES` table would crash threads working with internal temporary tables used by ``ALTER TABLE``. Bug fixed :bug:`1113388`.
+
+ Selecting from :table:`GLOBAL_TEMPORARY_TABLES` table while running an online ``ALTER TABLE`` on a partitioned table in parallel could lead to a server crash. Bug fixed :bug:`1193264`.
+
+ :ref:`innodb_kill_idle_trx` feature could cause an assertion due to a race condition. Bug fixed :bug:`1206008`.
+
+ ``libmylsqclient_16`` symbols were missing in |Percona Server| shared library package on *RHEL*/*CentOS* 7. Bug fixed :bug:`1420691`.
+
+ Prepared statements in stored procedures could crash :ref:`response_time_distribution`. Bug fixed :bug:`1426345`.
+
+ When variable :variable:`innodb_corrupt_table_action` is set to ``Warn/Salvage`` then loop to estimate stats makes a safe exit if it encounters a corrupt page. This safe exit path failed to close (commit) and open (started) mtr which would then lead to an issue while closing transaction. Bug fixed :bug:`1426610`.
+
+ Table API was using stale reference of table to check if table is corrupted. Bug fixed :bug:`1433197`.
+
+ Fixed upstream bug :mysqlbug:`77011` which caused a hanging ``System Lock`` when executing ``FLUSH TABLE ... FOR EXPORT``. Bug fixed :bug:`1454441`.
```

Comment:
"System Lock" is a "State" value in processlist output for the hanged table, this has to be clarified or it's not clear what a hanging system lock is.